src/test/FuzzerImpl.cpp

```diff
@@ -529,7 +529,7 @@ attemptToApplyOps(LedgerTxn& ltx, PublicKey const& sourceAccount,
         auto endOpsInThisTx =
             std::distance(beginOpsInThisTx, end) <= MAX_OPS_PER_TX
                 ? end
-                : begin + MAX_OPS_PER_TX;
+                : beginOpsInThisTx + MAX_OPS_PER_TX;
```

Comment:
While this fixes the problem, I don't think it's the best approach. Why did this bug happen? It happened because both `begin` and `beginOpsInThisTx` exist. As long as they both exist, this bug can happen again.

Perhaps having both exist was an unavoidable detail because you needed `begin` later. But you don't, so you might as well just remove `beginOpsInThisTx` entirely and use `begin`.

Reply:
Agreed, that's significantly more elegant.  Done.